cmd/nginx-ingress/flags.go

```diff
@@ -203,8 +201,9 @@ var (
 
 	enableDynamicSSLReload = flag.Bool(dynamicSSLReloadParam, true, "Enable reloading of SSL Certificates without restarting the NGINX process.")
 
-	enableTelemetryReporting = flag.Bool("enable-telemetry-reporting", true, "Enable gathering and reporting of product related telemetry.")
-	telemetryReportingPeriod = flag.String("telemetry-reporting-period", "24h", "Sets a telemetry reporting period.")
+	enableTelemetryReporting   = flag.Bool("enable-telemetry-reporting", true, "Enable gathering and reporting of product related telemetry.")
```

Comment:
docs for cli https://github.com/nginxinc/kubernetes-ingress/blob/main/docs/content/configuration/global-configuration/command-line-arguments.md